packages/ui/components/scrollable/ScrollableArea.tsx

```diff
@@ -35,7 +35,7 @@ const ScrollableArea = ({ children, className }: PropsWithChildren<{ className?:
         className // Pass in your max-w / max-h
       )}>
       {children}
-      {isOverflowingY && <div style={overflowIndicatorStyles} data-testid="overflow-indicator" />}
```

Comment:
Why was this removed? We get a warning of unused styles. If it's not needed then let's remove it completely.

Reply:
![Screenshot 2024-04-25 at 1 58 37 PM](https://github.com/calcom/cal.com/assets/32706411/6d17f026-26b6-4f1b-a7ae-bb7e46ec91f0)
it adds this shadow at the top, i think we can remove this

Reply:
removed 👍